src/MagnumPlugins/TinyGltfImporter/Test/TinyGltfImporterTest.cpp

```diff
@@ -2517,6 +2519,56 @@ void TinyGltfImporterTest::materialTexCoordFlip() {
     }), TestSuite::Compare::Container);
 }
 
+void TinyGltfImporterTest::materialTextureCoordinateSets() {
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("TinyGltfImporter");
+    CORRADE_VERIFY(importer->openFile(Utility::Directory::join(TINYGLTFIMPORTER_TEST_DIR,
+        "material-texcoord-sets.gltf")));
+    {
+        std::ostringstream out;
+        Error redirectError{&out};
+
+        CORRADE_VERIFY(!importer->material(0));
+        CORRADE_VERIFY(!importer->material(1));
+        CORRADE_VERIFY(!importer->material(2));
+        CORRADE_VERIFY(!importer->material(3));
+        CORRADE_COMPARE(out.str(),
+            "Trade::TinyGltfImporter::material(): multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them\n"
+            "Trade::TinyGltfImporter::material(): multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them\n"
+            "Trade::TinyGltfImporter::material(): multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them\n"
+            "Trade::TinyGltfImporter::material(): multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them\n");
+    }
```

Comment:
I would split the first part into a separate test, `materialTextureCoordinateSetsDefault()`